AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/KMSKeystoreOperations.dfy

```diff
@@ -291,6 +291,111 @@ module {:options "/functionSyntax:4" } KMSKeystoreOperations {
     return Success(reEncryptResponse);
   }
 
+  method VerifyViaDecryptEncryptKey(
```

Comment:
Praise: this is great.  
Having a KMS Configuration means we can use it for VersionKey in the future.  
You have raised the bar.